src/notifications/notificationsManager.ts

```diff
@@ -77,6 +77,10 @@ export class NotificationsManager {
 		return this._notifications.get(key);
 	}
 
+	public removeNotification(key: string): void {
+		this._notifications.delete(key);
```

Comment:
I don't think it will have any negative impact now, but it would be good to fire the change event if the removed notification existed in the notification map.

Reply:
Right thanks for the feedback!